creator-node/test/blacklistManager.test.js

```diff
@@ -6,24 +6,29 @@ const redis = require('../src/redis')
 
 const { getApp } = require('./lib/app')
 const { getLibsMock } = require('./lib/libsMock')
+const { restartBlacklistManager } = require('./lib/blacklistManager')
 
-describe('test blacklistManager', () => {
-  let server, libsMock, userId
+const DUMMY_CID = 'QmYkZMB1cdo8k1Lizco4YyiAMfUwCn9yUQaJn7T274uc5z'
```

Comment:
looks like most of the changes in this file are just reformatting and like small things like moving CID to DUMMY_CID. are there any test changes here?

Reply:
yup! just the const rename + using the `restartBlacklistManager()` fn